common/setups/hybrid/base_system.py

```diff
@@ -0,0 +1,165 @@
+__all__ = ["BaseSystem"]
+
+import itertools
+import sys
+from typing import Dict, List, Optional, Type
+
+# -------------------- Sisyphus --------------------
+
+import sisyphus.global_settings as gs
+import sisyphus.toolkit as tk
+
+# -------------------- Recipes --------------------
+
+import i6_core.am as am
+import i6_core.corpus as corpus_recipes
+import i6_core.meta as meta
+import i6_core.rasr as rasr
+
+from .util import HybridDataInput
+
+# -------------------- Init --------------------
+
+Path = tk.setup_path(__package__)
+
+# -------------------- System --------------------
+
+
+class BaseSystem(meta.System):
+    """
+    - 3 corpora types: train, dev and test
+    - only train corpora will be aligned
+    - dev corpora for tuning
+    - test corpora for final eval
+
+    to create beforehand:
+    - corpora: name and i6_core.meta.system.Corpus
+    - lexicon
+    - lm
+
+    settings needed:
+    - am
+    - lm
+    - lexicon
+    - feature extraction
+    - linear alignement
+    - monophone training
+    - monophone recognition
+    - triphone training
+    - triphone recognition
+    - vtln training
+    - vtln recognition
+    - sat training
+    - sat recognition
+    - vtln+sat training
+    - vtln+sat recognition
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        self.crp["base"].python_home = gs.RASR_PYTHON_HOME
```

Comment:
These settings are not necessary strictly speaking. Esp. if the RASR is compiled with system python it is fine to not set them and to not have `gs.RASR_PYTHON_HOME` set. I suggest to add somthing like
```if hasattr(gs, "SPRINT_PYTHON_HOME") and gs.SPRINT_PYTHON_HOME is not None:```

(or maybe we want to set `None` in this case?

Reply:
good point.

the default is None